### PR TITLE
RUN-1200-Update feature API doc for version 42

### PR DESCRIPTION
**Pull request by ehe-pd**
Update the API documents to reflect the change to feature query API.

### CODE REVIEW

**Comment by fdevans** [comment]
If this is related to the 4.9.0 release please do PR against the `4.9.0` branch so the docs are released at the proper time.

File: docs/api/rundeck-api-versions.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ Changes introduced by API Version number:
 **Deprecation**
 * API versions below `{{{ apiDepVersion }}}` are *deprecated*.  Clients using earlier versions should upgrade to use `{{{ apiDepVersion }}}` as the minimum version before release `{{{ apiDepRelease }}}` to avoid errors.
 
+**Version 42**:
+
+* Endpoints promoted out of incubating status. These endpoints no longer contain `incubating/` in the endpoint URL, and now require API version 42 minimum.
+    * [`GET /api/42/feature/[featureName]`][/api/42/feature/[featureName]] - Query system feature status.
```
Comment (gschueler) [inline docs/api/rundeck-api-versions.md]:
```suggestion
    * [`GET /api/42/feature/[featureName]`][/api/42/feature/\[featureName\]] - Query system feature status. 
```

File: docs/api/rundeck-api.md [hunk 2]
```diff
@@ -7834,6 +7834,70 @@ Content-Type: `application/json`
...
+### Get A System Feature Status
+
+
+Return the information about if a specific system feature is enabled or not.
+
+**Request:**
+
+    GET /api/42/feature/[featureName]
+
+The `featureName` parameter should be the feature's configuration name without `rundeck.feature.` prefix and `.enabled` surfix. E.g. For configuration name `rundeck.feature.runner.enabled`, the value of `[featureName]` should be `runner`
```
Comment (gschueler) [inline docs/api/rundeck-api.md]:
```suggestion
The `featureName` parameter should be the feature's configuration name without `rundeck.feature.` prefix and `.enabled` suffix. E.g. For configuration name `rundeck.feature.runner.enabled`, the value of `[featureName]` should be `runner`  
```

File: docs/api/rundeck-api.md [hunk 3]
```diff
@@ -7885,6 +7949,9 @@ Content-Type: `application/json`
 
 * `GET` [Execution Query Metrics](#execution-query-metrics)
 
+[/api/V/feature/[featureName]]
+* `GET` [System feature on/ff status](#system-feature)
```
Comment (gschueler) [inline docs/api/rundeck-api.md]:
```suggestion
* `GET` [System feature on/off status](#system-feature) 
```

File: docs/api/rundeck-api.md [hunk 4]
```diff
@@ -7885,6 +7949,9 @@ Content-Type: `application/json`
 
 * `GET` [Execution Query Metrics](#execution-query-metrics)
 
+[/api/V/feature/[featureName]]
```
Comment (gschueler) [inline docs/api/rundeck-api.md]:
```suggestion
[/api/V/feature/\[featureName\]][]
```

**Review by gschueler — CHANGES_REQUESTED** [review]
the file api-index-links.md also needs to be updated to include the link definition for `[/api/V/feature/\[featureName\]]` 

e.g. 

```md
[/api/V/feature/\[featureName\]]: /api/rundeck-api.html#system-feature
```

**Review by fdevans — APPROVED** [review]
Approved

**Review by gschueler — APPROVED** [review]

**Pull request merged**